### PR TITLE
1065311 cameraimagecapabilities

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Request for review

### CODE REVIEW

**Review by laujan — COMMENTED** [review]
@Rajeshwari-v, please update the TOC to include the new file.

**Comment by lokimeyburg** [comment]
1. Can we add a blurb or "note" about this only being for mobile near the top of the documentation?
2. Can we add a link from the `tabs/how-to/native-device-permissions` documentation to this documentation on how the "media" property is handled?
3. Finally, I noticed two images attached to this PR. Can we remove the duplicate and perhaps show one of the images in the documentation?

File: msteams-platform/concepts/build-and-test/enable-camera-image-capabilities.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          }
+      }
+      if (error) {
+          if (error.message) {
+            alert(" ErrorCode: " + error.errorCode + error.message);
+          } else {
+            alert(" ErrorCode: " + error.errorCode);
+          }
+      }
+      });
```
Comment (lokimeyburg) [inline msteams-platform/concepts/build-and-test/enable-camera-image-capabilities.md:117]: Formatting of the closing bracket has unnecessary spaces/tabs in the front of it. Consider removing the tab/spaces entirely.

**Comment by lokimeyburg** [comment]
* We should include screenshots in the documentation to help illustrate to developers what this API looks like when called?
* It's kind of weird to have two docs about device permissions in completely the different locations in the hierarchy (ie: `tabs > how-to` and under `Concepts > device permissions > mobile`. I feel like we should merge these two. Since this only applies to `tabs` perhaps we should move the documentation under there? ie:

```
tabs - embedded web experiences/
└─ how to guides/
      └─ device capabilities
           ├─ request device permissions 
           └─ enable-mobile-camera-image-permissions
```

**Comment by lokimeyburg** [comment]
LGTM 🐿

**Pull request merged**